mysql-test/suite/rpl_encryption/t/rpl_transaction_before_commit_failure-master.opt

```diff
@@ -0,0 +1 @@
+$RPL_OBS_EXAMPLE_OPT --early-plugin-load=$KEYRING_PLUGIN_DIR/$KEYRING_PLUGIN --keyring_file_data=$MYSQL_TMP_DIR/mysecret_keyring
```

Comment:
Merge to cnf

Reply:
Removed cnf file and added slave opt file.